src/App.js

```diff
@@ -169,6 +169,12 @@ export class ResourceEditor extends React.Component {
     }
   };
 
+  setLoading = (isLoading) => {
+    this.setState({
+      ui: { ...this.state.ui, loading: isLoading },
+    });
+  };
```

Comment:
Maybe you don't need to pass parameters here, you can use something like this to change the state:
```
setLoading = () => {
   const { loading } = this.state;
    this.setState({
      ui: { ...this.state.ui, loading: !loading  },
    });
  };
```

Reply:
I see your point, but in that case it would be more like a toggle. I just wanted to be able to explicitly specify the state regardless current state.